src/ripple/ledger/impl/View.cpp

```diff
@@ -391,7 +391,13 @@ transferRate (ReadView const& view,
     auto const sle = view.read(keylet::account(issuer));
 
     if (sle && sle->isFieldPresent (sfTransferRate))
-        return Rate{ sle->getFieldU32 (sfTransferRate) };
+    {
+        auto rate = (*sle)[sfTransferRate];
+        uint32_t maxRate = 2 * QUALITY_ONE;
+        if (view.rules().enabled(fix1201))
+            rate = std::min(rate, maxRate);
```

Comment:
Tiny nit: consider reformulating these lines as:
```
        auto rate = (*sle)[sfTransferRate];
        if (view.rules().enabled(fix1201))
        {
            constexpr uint32_t maxRate = 2 * QUALITY_ONE;
            rate = std::min(rate, maxRate);
        }
        return Rate{rate};
```
The `constexpr` assures you that the calculation happens at compile time.  Moving `maxRate` inside the braces limits its scope to the minimum necessary.